lib/oli_web/live/delivery/remix_section.ex

```diff
@@ -1047,6 +1047,8 @@ defmodule OliWeb.Delivery.RemixSection do
 
   defp transform_section_pages(section_pages) do
     section_pages
+    # Sort by title to ensure it matches the default order in the "all pages" authoring view
+    |> Enum.sort_by(fn rev -> rev.title end, :asc)
```

Comment:
I believe this would not be necessary and would even break with the logic of the sorting, since the database query we perform to obtain the data already sorts it there, and this `Enum.sort_by` would override the sorting we already have.
I believe it is not necessary here and we should remove it.

Reply:
Makes sense, since the "all pages" table allows user to change the sorting order. I will remove this. I don't think it is necessary for the all pages to match what is listed in the remix